docs/content/latest/yugabyte-cloud/cloud-quickstart/qs-data.md

```diff
@@ -166,6 +92,8 @@ You can now create a database and load some sample data.
     );
     ```
 
+## Load data
+
 1. Load some data into the products table by running the following commands.
```

Comment:
```suggestion
1. Load some data into the `products` table by running the following commands.
```

Reply:
Can you also highlight all the _table_ names in this load step. It makes a difference to be able to just see the table name since there are a couple of them.